src/components/pages/StudyCase/EditStudyCasePage.jsx

```diff
@@ -391,19 +403,21 @@ class EditStudyCasePage extends React.Component {
       id: this.props.studyCaseDetail.id,
       data: {
         ...this.form,
-        city_ids: [city.value],
+        city_ids: city && city.value ? [city.value] : [],
         category_id,
         // eslint-disable-next-line no-underscore-dangle
         impacts_attributes,
         // eslint-disable-next-line no-underscore-dangle
-        project_bmes_attributes: project_bmes_attributes.filter(pbme => !pbme.id || pbme.edited || pbme._destroy),
+        project_bmes_attributes: project_bmes_attributes // eslint-disable-next-line no-underscore-dangle
+          .filter(pbme => !pbme.id || pbme.edited || pbme._destroy)
+          .map(bme => ({
+            ...bme,
+            bme_id: bme.category_id
```

Comment:
Names don't add up. Why are we storing a Category's id on something named `bme_id`?